modules/javafx.graphics/src/test/java/test/com/sun/javafx/css/FixedCapacitySetTest.java

```diff
@@ -0,0 +1,225 @@
+package test.com.sun.javafx.css;
```

Comment:
The Copyright is missing from this class and the `if` and `for` clauses have no space after.
I think we usually do that, still minor of course. Tests are looking good to me!

Reply:
oops I missed that!  I think we must include the standard copyright header per OCA agreement.

Reply:
I always forget the header, thanks, I added it!